requirements.txt

```diff
@@ -5,3 +5,4 @@ pre-commit==2.7.1
 pytest==6.0.1
 hypothesis>=5.37.1
 pytest-cov>=2.10.1
+numpy>=1.19.2
```

Comment:
I don't think using ">=" is a good practice because we don't know if every update in this lib will be backwards compatible. Because of that, I believe that we should use a specific version of all libs using "==".